package.json

```diff
@@ -147,5 +147,8 @@
   "volta": {
     "node": "18.0.0",
     "npm": "8.6.0"
+  },
+  "dependencies": {
+    "jotai": "^2.2.2"
```

Comment:
Are there any issues with using React context instead that we must use another dependency?

My concern is bloating our peer dependencies with tools that aren't **_must-haves_** for core functionality.